jdk/test/java/awt/font/GlyphVector/MultiSlotFontTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8240756
+ * @summary Non-English characters are printed with wrong glyphs on MacOS
+ * @modules java.desktop/sun.java2d java.desktop/sun.java2d.loops java.desktop/sun.font
```

Comment:
Note that the test uses the @module tag which is noop for jdk8. The policy about this flag in jdk8 is [unclear](https://github.com/openjdk/jdk8u-dev/pull/405#discussion_r1445510434).

Reply:
I'd say remove it, but I'm not surprised some have slipped through when the patch is otherwise clean.